<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inscription</class>
 <widget class="QMainWindow" name="Inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscription</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="17" column="1">
      <widget class="QPushButton" name="boutton_Compte">
       <property name="text">
        <string>Créer Compte</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_eMail"/>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEdit_Pseudo">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mdp :</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QGroupBox" name="mailValide">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="buttonmailvalide">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>eMailvalidé</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="buttonmailInvalide">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>111</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>eMailNonvalidé</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QPushButton" name="retourAccueil">
       <property name="text">
        <string>Accueil</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Nom"/>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>eMail :</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="lineEdit_Mdp">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Prenom :</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_Prenom">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pseudo :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Nom</tabstop>
  <tabstop>lineEdit_Prenom</tabstop>
  <tabstop>lineEdit_eMail</tabstop>
  <tabstop>lineEdit_Pseudo</tabstop>
  <tabstop>lineEdit_Mdp</tabstop>
  <tabstop>boutton_Compte</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
